packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -219,6 +219,14 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
     }
   }
 
+  public async disposeStreamView(userId?: string, options?: VideoStreamOptions | undefined): Promise<void> {
+    if (userId === undefined) {
+      await this.handlers.onDisposeLocalStreamView();
+    } else {
+      await this.handlers.onCreateRemoteStreamView(userId, options);
```

Comment:
Should this be `onDispose...` instead of `onCreate...`?

Reply:
I don't love this pattern where if you put in args it does something and if you dont put in args it does something completely diffferent.

I am curious if we should be keep these functions explicitly different?

Reply:
@alkwa-msft Thanks Alex, I will continue working on this topic for following PRs, so let's merge this sdk update first then we can have more discussions on this one